Source/missiles.cpp

```diff
@@ -2862,7 +2862,7 @@ void MI_LArrow(Missile &missile)
 
 		missile_resistance rst = MissilesData[missile._mitype].mResist;
 		MissilesData[missile._mitype].mResist = MISR_NONE;
-		MoveMissileAndCheckMissileCol(missile, mind, maxd, true, true);
+		MoveMissileAndCheckMissileCol(missile, mind, maxd, true, false);

```

Comment:
The only thing that worries me at this point is how this change might exacerbate #1672.

Reply:
It will make it worse. If the logic is consistent between the two, that means trap elemental arrows haven't been dealing elemental damage most of the time, and now they will.